server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -979,13 +954,17 @@ func LookupInvocation(env environment.Env, ctx context.Context, iid string) (*in
 	eg, ctx := errgroup.WithContext(ctx)
 	eg.Go(func() error {
 		var scoreCard *capb.ScoreCard
-		// The cache ScoreCard is not stored in the table invocation, so we do this lookup
-		// after converting the table invocation to a proto invocation.
-		if ti.InvocationStatus == int64(inpb.Invocation_PARTIAL_INVOCATION_STATUS) {
-			scoreCard = hit_tracker.ScoreCard(ctx, env, iid)
-		} else {
-			if sc, err := readScoreCard(ctx, env, iid); err == nil {
-				scoreCard = sc
+		// When detailed stats are enabled, the scorecard is not inlined in the
+		// invocation.
+		if !hit_tracker.DetailedStatsEnabled() {
+			// The cache ScoreCard is not stored in the table invocation, so we do this lookup
+			// after converting the table invocation to a proto invocation.
+			if ti.InvocationStatus == int64(inpb.Invocation_PARTIAL_INVOCATION_STATUS) {
+				scoreCard = hit_tracker.ScoreCard(ctx, env, iid)
+			} else {
+				if sc, err := scorecard.Read(ctx, env, iid); err == nil {
+					scoreCard = sc
+				}
```

Comment:
we probably want to log this error rather than swallowing it

Reply:
Done